src/calculateRentalCost.js

```diff
@@ -3,8 +3,24 @@
  *
  * @return {number}
  */
+
+const LONG_TERM = 7;
+const MID_TERM = 3;
```

Comment:
maybe will better 🤔 ?
```suggestion
const SHORT_TERM = 3;
```